### PR TITLE
Add private attr readers

### PR DESCRIPTION
**Pull request by ksarunas**
@laurynas @astrauka @tomasv

### CODE REVIEW

File: lib/interactor/initializer/attr_readers.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class Interactor::Initializer::AttrReaders
   def self.for(target_class, attributes)
     target_class.class_eval { attr_reader(*attributes) }
```
Comment (tomasv) [inline lib/interactor/initializer/attr_readers.rb]: Maybe:

``` ruby
target_class.class_eval do
  attr_reader(*attributes)
  attributes.each { |attribute| private(attribute) }
end
```

Comment (tomasv) [inline lib/interactor/initializer/attr_readers.rb]: Or even:

``` ruby
target_class.class_eval do
  attributes.each do |attribute|
    attr_reader(attribute)
    private(attribute)
  end
end
```

**Comment by ksarunas** [comment]
updated @tomasv

**Comment by tomasv** [comment]
👍

**Comment by laurynas** [comment]
👍

**Pull request merged**